docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,341 @@
+:url-diagram: https://github.com/asciidoc/asciidoc-diagram
+:url-migrate: https://github.com/asciidoc/docs/migration
+:url-recommended: http://asciidoctor.org/docs/asciidoc-recommended-practices
+:url-diffs: http://asciidoctor.org/docs/asciidoc-asciidoctor-diffs/
+== Migrating from {ap} to {ad}
+
+The purpose of this section is to get documents that converted correctly using {ap} to produce the same output using {ad}.
+The differences are very small so that most documents will need very few or no changes.
+Once you have done the basic changes, this section also describes what to change to make use of all the new features in {ad}.
+
+=== Command line
+
+{ad} was designed from the outset to be a drop-in replacement for {ap}. 
+For most documents therefore you can just replace the {ap} call:
+
+`{ad} mydocument.adoc`
+
+with this {ad} call:
+
+`{ad} -a compat-mode mydocument.adoc`
+
+If this doesn't work for you, check out the differences below, particularly <<migrate-deprecated>> and <<migrate-extensions>>.
+
+==== internationalisation
+
+// ref ap UG, not checked
+{ap} has built-in .conf files so that it can translate captions just by setting `-a lang=<language code>`. 
+See <<language-support>> for the {ad} equivalent.
+
+==== Configuration files
+
+{ad} does not use .conf files or filters, so, `--conf-file`, `-dump-conf`, and `--filter` are not implemented. 
+
+==== Themes
+
+// ref ap UG. Actually seems to set JS as well. OK to ignore?
+{ap} themes are just stylesheets, so replace `--theme` with `-a stylesheet=<stylesheet>`.
+
+==== Compact
+
+// ref ap UG. 
+{ap} `--compact` reduces line spacing in lists. 
+To achieve the same in {ad}, use a custom stylesheet.
+
+==== Doctest
+
+// ref ap UG. 
+{ap} `--doctest` runs units tests. 
+See GitHub for how to run the {ad} unit tests.
+
+==== Default html backend
+
+{ap}: xhtml (version 1.1).
+ad}: html5. 
+Use `--backend xhtml` to replicate {ap} behaviour.
+
+=== Changed syntax
+
+// Simplified from migration guide
+These changes are non backward-compatible, but if you set the the `compat-mode` attribute, {ad} will accept the {ap} syntax.
+For the long term you should convert to the {ad} syntax.
+
+[cols="1a,1a,1a"]
+|====
+|Feature |{ap} or {ad} with `compat-mode` |{ad}
+
+|_italic text_
+|pass:['italic text']
+|pass:[_italic text_]
+
+|`monospaced text`
+|pass:[+monospaced text+]
+|pass:[`monospaced text`]
+
+// Check - I may be confused
+|Prevent attribute replacement
+|pass:[`{asciidoc-version}`]
+|pass:[`+{asciidoc-version}+`]
+
+|"`double quotes`"
+|pass:[``double quotes'']
+|pass:["`double quotes`"] or enter them directly
+
+|'`single quotes`'
+|pass:[`single quotes']
+|pass:['`single quotes`'] or enter them directly
+
+|Document title
+|
+[source]
+----
+Title
+=====
+----
+|
+[source]
+----
+= Title
+----
+
+|====
+
+// I am sure that this was good idea at the time, but...
+CAUTION: Do NOT use the underline-with-equals form for the document title; this implicitly sets `compat-mode`.
```

Comment:
I don't think we should say "do NOT use". Instead, we should say "if you use it, then it implicitly sets `compat-mode`. The correct name for this form is "Setext-style" (not underline-with-equals).